demo/loadFeatures.py

```diff
@@ -22,38 +24,38 @@ def eachFeature():
         pass
 
 
-def loadFeatures(esHost, featureSetName='movie_features'):
+def loadFeatures(featureSetName='movie_features'):
     featureSet = {
         "featureset": {
             "name": featureSetName,
             "features": [feature for feature in eachFeature()]
         }
     }
     path = "_ltr/_featureset/%s" % featureSetName
-    fullPath = urljoin(esHost, path)
+    fullPath = urljoin(ES_HOST, path)
     print("POST %s" % fullPath)
     print(json.dumps(featureSet, indent=2))
     head = {'Content-Type': 'application/json'}
-    resp = requests.post(fullPath, data=json.dumps(featureSet), headers=head)
+    resp = requests.post(fullPath, data=json.dumps(featureSet), headers=head, auth=ES_AUTH)
     print("%s" % resp.status_code)
     print("%s" % resp.text)
 
 
 
-def initDefaultStore(esHost):
-    path = urljoin(esHost, '_ltr')
+def initDefaultStore():
+    path = urljoin(ES_HOST, '_ltr')
     print("DELETE %s" % path)
-    resp = requests.delete(path)
+    resp = requests.delete(path, auth=ES_AUTH)
     print("%s" % resp.status_code)
     print("PUT %s" % path)
-    resp = requests.put(path)
+    resp = requests.put(path, auth=ES_AUTH)
     print("%s" % resp.status_code)
 
 
 
 if __name__ == "__main__":
     from time import sleep
-    esHost='http://localhost:9200'
-    initDefaultStore(esHost)
+    from utils import ES_HOST
+    initDefaultStore(ES_HOST)
```

Comment:
This function doesn't take arguments anymore, it gets ES_HOST directly. Similarly the import on line 58 is unnecessary since it's also imported on line 3. Chances are good i wrote that bug too :)